website/documentation/050-tutorials/content/howto/insecure-configuration/_index.md

```diff
@@ -13,48 +13,87 @@ aliases: ["readmore/insecure"]
 
 {{% howto_img "" "./images/teaser.svg" %}}
 
-## Introduction
-In the last summer, the [Gardener team](https://github.com/gardener/gardener) worked on assessing the security of 
-different Gardener managed Kubernetes installations. **The goal of this ongoing work is to increase the security of 
-all Gardener stakeholders in the Open Source community.** Following the Gardener architecture, the control plane of a 
-Gardener managed shoot cluster resides in the corresponding seed cluster. This is a 
-[Control-Plane-as-a-Service](https://kubernetes.io/blog/2018/05/17/gardener/#kubernetes-control-plane) with 
+## Increasing the Security of all Gardener Stakeholders
+In summer 2018, the [Gardener project team](https://github.com/gardener/gardener) asked [Kinvolk](https://kinvolk.io/) to execute several penetration tests in its role as third-party contractor. The goal of this ongoing work is to increase the security of
+all Gardener stakeholders in the open source community. Following the Gardener architecture, the control plane of a
+Gardener managed shoot cluster resides in the corresponding seed cluster. This is a
+[Control-Plane-as-a-Service](https://kubernetes.io/blog/2018/05/17/gardener/#kubernetes-control-plane) with
 a [network air gap](https://kubernetes.io/blog/2018/05/17/gardener/#network-air-gap).
 
 
-Along the way we found different kinds of security issues, for example due to misconfiguration or missing isolation, 
-as well as two problems with upstream Kubernetes which especially come into picture in the Control-Plane-as-a-Service 
+Along the way we found various kinds of security issues, for example, due to misconfiguration or missing isolation,
+as well as two special problems with upstream Kubernetes and its Control-Plane-as-a-Service
 architecture.
 
-## Sharing the findings
-From this experience, we’d like to share a few examples of security issues that could happen on a Kubernetes 
-installation and how to fix them. 
+## Major Findings
+From this experience, we’d like to share a few examples of security issues that could happen on a Kubernetes
+installation and how to fix them.
 
-Those examples cover the following scenarios:
+Alban Crequy [Kinvolk](https://kinvolk.io/) and Dirk Marwinski ([SAP SE](https://www.sap.com)) gave a presentation entitled [Hardening Multi-Cloud Kubernetes Clusters as a Service](https://kccncchina2018english.sched.com/event/H2Hd/hardening-multi-cloud-kubernetes-clusters-as-a-service-dirk-marwinski-sap-se-alban-crequy-kinvolk-gmbh) at KubeCon 2018 in Shanghai presenting some of the findings.
```

Comment:
Please put Kinvolk in parenthesis as you have done it with SAP

Reply:
Thanks Frederik - I added the missing brackets now with my last commit.